x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/list/policy_event_filters_list.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import uuid from 'uuid';
+import { getExceptionListItemSchemaMock } from '../../../../../../../../lists/common/schemas/response/exception_list_item_schema.mock';
+import {
+  AppContextTestRender,
+  createAppRootMockRenderer,
+} from '../../../../../../common/mock/endpoint';
+import { getPolicyEventFiltersPath } from '../../../../../common/routing';
+import { eventFiltersListQueryHttpMock } from '../../../../event_filters/test_utils';
+import { PolicyEventFiltersList } from './policy_event_filters_list';
+
+describe('Policy details event filters list', () => {
+  let policyId: string;
+  let render: () => Promise<ReturnType<AppContextTestRender['render']>>;
+  let renderResult: ReturnType<AppContextTestRender['render']>;
+  let history: AppContextTestRender['history'];
+  let mockedContext: AppContextTestRender;
+  let mockedApi: ReturnType<typeof eventFiltersListQueryHttpMock>;
+
+  beforeEach(() => {
+    policyId = uuid.v4();
+    mockedContext = createAppRootMockRenderer();
+    mockedApi = eventFiltersListQueryHttpMock(mockedContext.coreStart.http);
+    ({ history } = mockedContext);
+    render = async () => {
+      await act(async () => {
+        renderResult = mockedContext.render(<PolicyEventFiltersList policyId={policyId} />);
+        await waitFor(mockedApi.responseProvider.eventFiltersList);
+      });
+      return renderResult;
+    };
+
+    act(() => {
+      history.push(getPolicyEventFiltersPath(policyId));
+    });
+  });
+
+  it('should display a searchbar and count even with no exceptions', async () => {
+    mockedApi.responseProvider.eventFiltersList.mockReturnValue({
+      total: 0,
+      page: 1,
+      per_page: 10,
+      data: [],
+    });
+    await render();
+    expect(renderResult.getByTestId('policyDetailsEventFiltersSearchCount')).toHaveTextContent(
+      'Showing 0 exceptions'
+    );
+    expect(renderResult.getByTestId('searchField')).toBeTruthy();
+  });
+
+  it('should render the list of exceptions collapsed and expand it when clicked', async () => {
+    mockedApi.responseProvider.eventFiltersList.mockReturnValue({
```

Comment:
tip: `getFoundExceptionListItemSchemaMock` can return this data structure already done for you, for example: `getFoundExceptionListItemSchemaMock(3)` will return you this list with 3 exceptions inside.